spec/models/callback_modifier_spec.rb

```diff
@@ -23,6 +23,19 @@
             modifier.test_destroy
             expect(modifier.versions.last.reify).to be_flagged_deleted
           end
+
+          if active_record_gem_version.release >= Gem::Version.new("5")
+            context "when ActiveRecord::Base.belongs_to_required_by_default is enabled" do
+              it "should display a deprecation warning" do
+                allow(::ActiveRecord::Base).to receive(:belongs_to_required_by_default).
+                  and_return(true)
+                expect(::ActiveSupport::Deprecation).to receive(:warn).
+                  with(/paper_trail_on_destroy(:after) is incompatible with ActiveRecord/)
```

Comment:
Good catch on the parenthesis thing! I should have noticed that previous and I've fixed in an amended commit: https://github.com/airblade/paper_trail/pull/813/commits/0bf252eb4c22af36f111b0f8812d96b973c0ab2d

However, that doesn't seem to be the actual issue. If that were the problem, I would have seen the method called 1 time with a mismatched argument, but instead I'm seeing it called 0 times...